packages/html/src/index.ts

```diff
@@ -6,9 +6,7 @@ import { RollupHtmlOptions, RollupHtmlTemplateOptions } from '../types';
 
 const getFiles = (bundle: OutputBundle): RollupHtmlTemplateOptions['files'] => {
   const files = Object.values(bundle).filter(
-    (file) =>
-      file.type === 'chunk' ||
-      (typeof file.type === 'string' ? file.type === 'asset' : file.isAsset)
+    (file: any) => file.isEntry || !file.fileName.endsWith('.js')
```

Comment:
so this isn't exactly what we're after. we do want all files emitted in the build to be present in the result here.

Reply:
In that case, I think we can remove the filter altogether and rename `getFiles` to `groupFiles` ? What do you think?

Reply:
In the end I went with `getFilesByExtension`... Check out 233fe7c and let me know if that looks good. Thanks!

Reply:
that's not going to work for me. I commented on the main PR thread as to why